### DISCUSSION TITLE
How to automaticly execute action when condition is true

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#905] [Q&A]
Hi

mv-action is executed only on click of element,

```
<div mv-app="main">
      <div property="oldname"></div>
      <div property="newname"></div>

<button mv-action="setif(newname!=oldname,backup.newname,newname)">Copy</button>
</div>

<div mv-app="target">
     <div property="oldname"></div>
     <div property="newname"></div>
</div>
```

I have a usecase, where I would like to execute mv-action, if a value of one or multiple properties change
not only when I click on button, but each time property value changes by itself

Does someone knows how to achieve this?

PS: My usecase can't just ask **target** app to retrieve value from **main** app. 
My case needs the **main** app, to _send_ value to **target** app, only if properties values of main app changes

I wasn't able to do it with "setif" as it will need a click on button to execute

Maybe semething using 'onchange'? But I don't see how to make it work with mavo, as the change will be on the input, that mavo applyes on top of the div

**Comment by DmitrySharabin** [comment]
Hey there,

I'm not sure that I fully understand your use case, but here is what I came up with using Mavo API: https://codepen.io/dmitrysharabin/pen/LYdMVmo.

There is a risk that the way I propose it to do is NOT the one that one should use. I hope somebody can correct me if I'm violating anything.

**Comment by GalinhaLX** [comment]
Hi Dmitry

Thank you so much for your codepen :)

For 1 property it works, great

I tried to add multiple properties as you mentioned
`const properties = ["old_name","new_name"]; // All properties that need to be tracked`

in this case, it only push all values if I modify `property="old_name"` in last position
Does it need to have another eventlistener somewhere?

[modified codepen here](https://codepen.io/galinhalx/pen/zYWePNy)
